dockertest/subtestbase.py

```diff
@@ -168,6 +172,23 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
+        except IOError, anotherone:
+            raise xcept("Error reading %s: %s"
+                        % (cls.redhat_release_filepath,
+                           str(anotherone)))
```

Comment:
...similarly here. If these tests are part of standard docker-autotest runs, a user on <non-RH-CentOS-or-Fedora> might prefer a message such as "This doesn't look like a Red Hat system".